Dockerfile

```diff
@@ -44,27 +46,28 @@ WORKDIR /build
 COPY . .
 RUN make build
 
-FROM requirements
 
-ARG FFMPEG
+# FROM requirements
```

Comment:
I am wondering why here we need FROM requirements again and run `RUN make prepare-sources`, Can we merge them to the `make build` part together?

Reply:
Is it can run tasks in parallel sequence?

```
[+] Building 282.0s (19/22)                                                                                                                                                                                                                             
 => [requirements 3/9] RUN bash /tmp/library-scripts/debian.sh ""     && apt-get clean -y && rm -rf /var/lib/apt/lists/* /tmp/library-scripts                                                                                                     12.8s 
 => [requirements 4/9] COPY scripts/cu-blas.sh /tmp/library-scripts/                                                                                                                                                                               0.4s 
 => [requirements 5/9] RUN bash /tmp/library-scripts/cu-blas.sh "${BUILD_TYPE}" "1.0-1" "11 7"     && apt-get clean -y && rm -rf /var/lib/apt/lists/* /tmp/library-scripts                                                                         0.8s 
 => [requirements 6/9] COPY scripts/open-blas.sh /tmp/library-scripts/                                                                                                                                                                             0.4s
 => [requirements 7/9] RUN bash /tmp/library-scripts/open-blas.sh     && apt-get clean -y && rm -rf /var/lib/apt/lists/* /tmp/library-scripts                                                                                                      8.7s
 => [requirements 8/9] COPY scripts/stable-diffusion.sh /tmp/library-scripts/                                                                                                                                                                      0.5s
 => [requirements 9/9] RUN bash /tmp/library-scripts/stable-diffusion.sh     && apt-get clean -y && rm -rf /var/lib/apt/lists/* /tmp/library-scripts                                                                                             209.5s
 => [stage-2 1/6] COPY scripts/ffmpeg.sh /tmp/library-scripts/                                                                                                                                                                                     0.9s
 => [builder 1/3] WORKDIR /build                                                                                                                                                                                                                   0.7s
 => [builder 2/3] COPY . .                                                                                                                                                                                                                         1.0s
 => [stage-2 2/6] RUN bash /tmp/library-scripts/ffmpeg.sh "${FFMPEG}"     && apt-get clean -y && rm -rf /var/lib/apt/lists/* /tmp/library-scripts                                                                                                  1.4s 
 => [builder 3/3] RUN make build                                                                                                                                                                                                                  44.8s 
 => => # git clone --recurse-submodules https://github.com/mudler/go-stable-diffusion go-stable-diffusion                                                                                                                                               
 => => # Cloning into 'go-stable-diffusion'...                                                                                                                                                                                                          
 => => # Submodule 'ncnn' (https://github.com/Tencent/ncnn) registered for path 'ncnn'                                                                                                                                                                  
 => => # Submodule 'stable-diffusion' (https://github.com/EdVince/Stable-Diffusion-NCNN) registered for path 'stable-diffusion'                                                                                                                         
 => => # Cloning into '/build/go-stable-diffusion/ncnn'...                                                                                                                                                                                              
 => => # Cloning into '/build/go-stable-diffusion/stable-diffusion'...                                                                                                                                                                                  
 => [stage-2 3/6] WORKDIR /build                                                                                                                                                                                                                   0.6s 
 => [stage-2 4/6] COPY . .                                                                                                                                                                                                                         0.5s 
 => [stage-2 5/6] RUN make prepare-sources                                                                                                                                                                                                        43.1s 
 => => # git clone --recurse-submodules https://github.com/mudler/go-stable-diffusion go-stable-diffusion                                                                                                                                               
 => => # Cloning into 'go-stable-diffusion'...                                                                                                                                                                                                         
 => => # Submodule 'ncnn' (https://github.com/Tencent/ncnn) registered for path 'ncnn'                                                                                                                                                                 
 => => # Submodule 'stable-diffusion' (https://github.com/EdVince/Stable-Diffusion-NCNN) registered for path 'stable-diffusion'                                                                                                                        
 => => # Cloning into '/build/go-stable-diffusion/ncnn'...                                                                                                                                                                                             
 => => # Cloning into '/build/go-stable-diffusion/stable-diffusion'...                                                                                                                                                                                 

```